app/src/ui/index.tsx

```diff
@@ -138,6 +139,7 @@ dispatcher.registerErrorHandler(defaultErrorHandler)
 dispatcher.registerErrorHandler(upstreamAlreadyExistsHandler)
 dispatcher.registerErrorHandler(externalEditorErrorHandler)
 dispatcher.registerErrorHandler(openShellErrorHandler)
+dispatcher.registerErrorHandler(mergeConflictHandler)
```

Comment:
> should we be feature flagging any of this?

We could feature-flag this registration here, but given the new flow is only slightly different to what we had before I'm not hugely worried about it.